recipes-samples/images/rpb-weston-image.bb

```diff
@@ -15,6 +15,7 @@ CORE_IMAGE_BASE_INSTALL += " \
     sshfs-fuse hostapd iptables \
     alsa-utils-aplay gstreamer1.0-plugins-bad-meta gstreamer1.0-plugins-base-meta gstreamer1.0-plugins-good-meta \
     ${@bb.utils.contains("MACHINE_FEATURES", "optee", "optee-test optee-client", "", d)} \
+    ${@bb.utils.contains("MACHINE_FEATURES", "mali450", "mali450-userland", "", d)} \
```

Comment:
the way I solved that for meta-qcom is

MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'mesa-driver-msm', '', d)}"

i am not saying it's better.. but considering mali450 a MACHINE_FEATURES looks weird to me. maybe it's been discussed and agreed already, in which case it's fine..